finalPrep/annotations.js

```diff
@@ -3,40 +3,94 @@
 
 //Build a Bear
 function buildABear(name, age, fur, clothes, specialPower) {
+  //the function buildABear is declared with 5 parameters
```

Comment:
I'd say it is more common to place line by line annotations above the line of code, but you'll see both out there.